crates/cac_client/src/utils.rs

```diff
@@ -1 +1,29 @@
+use itertools::{self, Itertools};
+use serde_json::Value;
 pub mod core;
+
+pub fn json_to_sorted_string(v: &Value) -> String {
```

Comment:
let's expose this function from lib.rs of this crate and use it `context-aware-config` crate as well, delete the copy present in `context-aware-config`

lets make a conscious effort to avoid code repetition